components/brave_rewards/common/pref_names.cc

```diff
@@ -29,6 +29,7 @@ const char kUpholdAnonAddress[] =
     "brave.rewards.uphold_anon_address";
 const char kBadgeText[] = "brave.rewards.badge_text";
 const char kUseRewardsStagingServer[] = "brave.rewards.use_staging_server";
+const char kWalletType[] = "brave.rewards.wallet_type";
```

Comment:
Nit: "wallet" is pretty ambiguous for us, can we use "external_wallet_type" here?

Reply:
resolved